features/landing_page.feature

```diff
@@ -0,0 +1,103 @@
+      | emma@mutts.com | 2017-10-1  | 2018-02-27  | member_fee   | betald | none    |                |
+
+  Scenario: After login, Admin still sees new memberships on their landing page
+    Given I am logged in as "admin@shf.se"
+    When I am on the "landing" page
+    Then I should see t("shf_applications.index.title")
+
+  @selenium @time_adjust
+  Scenario: After login, Member sees correct instructions
+    Given the date is set to "2017-01-01"
+    And I am logged in as "emma@mutts.com"
+    When I am on the "user instructions" page
+    Then I should not see t("shf_applications.index.title")
+    And I should see t("shf_applications.information.member.using_the_h_mark")
+    And I should not see t("menus.nav.users.apply_for_membership")
+    And I should not see t("shf_applications.information.waiting_for_approval.in_process")
+
+  Scenario: After login, User sees how to apply etc
+    Given I am logged in as "new@mutts.com"
+    When I am on the "user instructions" page
+    Then I should not see t("info.logged_in_as_admin")
+    And I should not see t("shf_applications.information.member.using_the_h_mark")
+    And I should not see t("shf_applications.information.waiting_for_approval.in_process")
+    And I should see t("menus.nav.users.apply_for_membership")
+
+  Scenario: Visitor does not see instructions
+    Given I am Logged out
+    When I am on the "user instructions" page
+    Then I should not see t("info.logged_in_as_admin")
+    And I should not see t("shf_applications.information.member.using_the_h_mark")
+    And I should not see t("shf_applications.information.waiting_for_approval.in_process")
+    And I should not see t("menus.nav.users.apply_for_membership")
+
+  @selenium @time_adjust
+  Scenario: After login, Member sees correct instructions about member fee
+    Given the date is set to "2018-03-01"
+    And I am logged in as "emma@mutts.com"
+    When I am on the "user instructions" page
+    Then I should see t("shf_applications.information.check_payment_status.m_expires")
+    When I click on t("menus.nav.members.pay_membership")
+    Then I should be on the "user details" page for "emma@mutts.com"
+
+  @selenium @time_adjust
+  Scenario: After login, Member sees correct instructions about H branding
+    Given the date is set to "2018-02-01"
+    And I am logged in as "emma@mutts.com"
+    When I am on the "user instructions" page
+    Then I should see t("shf_applications.information.check_payment_status.h_expires")
+    And I should not see t("shf_applications.information.check_payment_status.m_expires")
+    And I should not see t("shf_applications.information.waiting_for_approval.in_process")
+    When I click on t("menus.nav.company.pay_branding_fee")
+    Then I should be on the "my company" page for "emma@mutts.com"
+
+  Scenario: After login, Member without previously paid Membership fee sees correct instructions
```

Comment:
Change the scenario heading to:
```
After login, user with accepted app but with no membership payment sees instructions to pay
```
That better reflects the situation of this test.